app/views/products/index.html.erb

```diff
@@ -11,7 +11,7 @@
         <li>Stock: <%= product.stock %> units</li>
         <li>Available: <%= product.available %></li>
         <li>User_id: <%= product.user_id %></li>
-        <img src= <%= product.photo_url %>
+        <img src= <%= product.photo_url %> alt=<% product.name %> height="200" width="200">
```

Comment:
I think for the alt text you need to use `<%=`, but I'm not sure.

Reply:
Done!